decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDeciderCli.java

```diff
@@ -103,6 +104,9 @@ public Bcl2fastqDeciderCli() {
         dryRunOpt = parser.acceptsAll(Arrays.asList("dry-run", "test"),
                 "Dry-run/test mode. Prints the INI files to standard out and does not submit the workflow.")
                 .withOptionalArg().ofType(Boolean.class).defaultsTo(false);
+        demultiplexSingleSampleModeOpt = parser.acceptsAll(Arrays.asList("enable-demultiplex-single-sample-mode"),
```

Comment:
I think the CLI option should be a little shorter. How about "demux-single-sample"?